src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/main_corpus.inc.bkp

```diff
@@ -0,0 +1,237 @@
+
```

Comment:
@glpuga a `vim` backup?

Reply:
nop, I'm not a VIM user.

This is just a manual back up of the code I had to remove to be able to generate the pdf skipping the failing timem data. Loading the timem data fails in the very first loading of the data, so the try-excepts were not enough to get through this failure mode.

This needs to be restored once I figure why timem data is corrupted.

Reply:
done.